sentry-android-core/src/main/java/io/sentry/android/core/AppStartState.java

```diff
@@ -44,14 +44,14 @@ void setAppStartEnd(final long appStartEndMillis) {
   }
 
   @Nullable
-  synchronized Long getAppStartInterval() {
+  public synchronized Long getAppStartInterval() {
```

Comment:
Can we instead have public getter methods to separately get the `appStartMillis` and `appStartEndMillis` for RN? We would need the separate timestamps to compute RN's app start with own JS's launch time.